### PR TITLE
feature/admin can export orders to a csv

### PR DESCRIPTION
**Pull request by kajawitek**
Hi Nerds 🤓

I added a new feature. Admin can now export orders to CSV file.

AC:

- There is a button that allows an admin to download all orders
- The CSV includes all order fields
- There is a test that covers exporting orders

### Heroku review app:

**Employee:**
https://emp-kw-pr-27.herokuapp.com/
Login: email1@test.com
Password: Password

**Admin:**
https://emp-kw-pr-27.herokuapp.com/admins/
Login: admin@test.com
Password: Password1

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+
+  def self.to_csv
+    orders = all
+    CSV.generate do |csv|
+      csv << column_names
+      orders.each do |order|
+        csv << order.attributes.values_at(*column_names)
+      end
+    end
+  end
```
[the remark this answers is not in the document]
Comment (kajawitek) [reply app/models/order.rb]: You're right, it makes more sense. I corrected it - now the admin can see all information from the index view and the orders table 🙂

File: app/models/order.rb [hunk 2]
```diff
@@ -1,6 +1,17 @@
+require 'csv'
```
[the remark this answers is not in the document]
Comment (kajawitek) [reply app/models/order.rb]: I don't have it in the current version of the order model. I'll keep that in mind in the future 🙂

File: app/views/admins/orders/orders.csv.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<%=raw CSV.generate_line(["Status", "Order ID", "Reward ID", "Reward title", "Reward description", "Order creation time", "Update time", "Purchase price", "Employee ID", "Employee email"]) -%>
+<%- @orders.each do |order| -%>
+  <%=raw CSV.generate_line([order.delivery_status, 
+                         order.id,
```
Comment (ciejj) [inline app/views/admins/orders/orders.csv.erb:4]: formatting issue all `order.` should be aligned

**Review by ciejj — APPROVED** [review]
👍

File: app/views/admins/orders/orders.csv.erb (hunk 3 above)
Comment (kajawitek) [reply app/views/admins/orders/orders.csv.erb]: corrected 🙂

**Comment by kajawitek** [comment]
@piowit @ciejj - thank you! 🙂

**Pull request merged**